src/scss/_variables.scss

```
@@ -17,8 +17,9 @@ $color-focus: transparentize($color-highlight, .8);
 $color-background-menu-title-overlay: rgba(0, 0, 0, .25) !default;
 $color-background-menu: #2e2e2e !default;
 $color-background-menu-hover-overlay: rgba(255, 255, 255, .07) !default;
-$color-background-seek-circle: rgba(124, 124, 124, .35) !default;
+$color-background-seek-circle: rgba(255, 255, 255, .35) !default;
 $color-shadow-seek-label: 0 0 30px 0 rgba(0, 0, 0, .75) !default;
+$color-touch-control-overlay-background: rgba(0, 0, 0, .5) !default;
```

Nit: I'd honestly make the overlay completely transparent. I see for example YouTube does that.
Cause I don't see the benefit of showing it to the user

```suggestion
$color-touch-control-overlay-background: rgba(0, 0, 0, 0) !default;
```

---

The difference between touch input and non touch input devices is that the main pause/play toggle action is in the middle of the screen. The pause button is visible on mobile while it's not visible on web. When we have a very bright frame behind it, it's hard to see, hence I decided to add this background.